### DISCUSSION TITLE
Welcome to `react-router` Discussions!

### DISCUSSION THREAD

**Discussion by chaance** [remix-run/react-router#8006] [General]
## Welcome to `react-router` discussions! 👋
  We’re using GitHub Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about 🤔
  * Share ideas 💡
  * Engage with other community members 🤝
  * Welcome others and are open-minded—remember that this is a community we build together 💪

To get started, drop an introduction below! Let us know more about who you are, what you're building with `react-router` and how you'd like to be contribute or participate here on GitHub.

**Comment by chaance** [comment]
I suppose I should go first!

Hi 👋 My name is Chance, and I've been building websites and web software for several years. Like many elder millennials I started dabbling with CSS in the MySpace days, and eventually I moved to customizing WordPress themes until I learned enough PHP to be dangerous. 

Soon enough I found myself picking up JavaScript and I got hooked on the frontend. Shortly after it was released I heard about this new "reactive" framework, and the rest is history. I worked for a few years as a full-time freelancer building React Apps and consulting for various teams, until eventually I got hooked up with Michael and Ryan who hired me to work on [Reach UI](https://reach.tech/) as a member of the [React Training](https://reacttraining.com/) team.

Since then I spent a bit of time working with [Modulz](https://www.modulz.app/), teaching, and contracting before I started working with Remix. My first priority on the team is getting React Router 6 across the finish line, and I'm fortunate to have inherited a strong and passionate community of folks who have helped to get us were we are today. I look forward to working with you all to get it done 💪

A huge goal for me is to improve the default accessibility in both RR v5 and v6. In the early days I think it made sense for the router to stay hands-off, but we know a lot more these days about what it takes to make client-side routing better for everyone.

When I'm not booping around on the computer, I love skateboarding, running, climbing, hiking, surfing—pretty much anything outside that gets my blood flowing. I occasionally play drums and guitar, and I take my pup Lelu for lots of walks around San Diego. ☀️

**Comment by bogadrian** [comment]
My name is Adrian and I am a middle React developer. 
I had some problems with the lazy loading. There was some CSS chunk not loaded.
can't React Router come with a lazy loading feature packed in?

**Reply by Mostafa-Mohammadi** [reply]
I had the same problem ,

the solution is  you should import your Component in order , i mean first of All your general function  , then your CSS file , and the last one your Dynamic import

**Comment by parthjmistry** [comment]
I'm Parth. I am thinking react router should have variable support in the baseUrl

For example I should be able to define baseUrl as below
"/{tenant}" and I should be able to get tenant parameters using useParam hook.

**Comment by chaance** [comment]
Welcome folks! Just a note, if you have a feature idea I'd go ahead and create a separate discussion for it. That'll be much easier for us to track and make sure we don't miss anything.

**Comment by TathagatIVP** [comment]
How can we access props passed to the component  inside the loader function in react router 6.4

**Reply by machour** [reply]
Please open a new "Q&A" conversation
